_posts/2019-01-25-jesse-jiryu-davis.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "PyTN 2019 Speaker Series: A. Jesse Jiryu Davis"
+published: true
+categories: [blog, tech, conference]
+tags: [python, tech, conference, tennessee, nashville]
+---
+A. Jesse Jiryu Davis is one of our two keynote speakers presenting a talk at PyTN 2019. We are thrilled to have Jesse as a speaker, and share the enthusiasm for Python and PyTN 2019 below!
+
+![Jesse Jiryu Davis](/static/img/2019-speakers/jesse_jiryu_davis.jpg)
```

Comment:
The path for those pictures should have changed from `2019-speakers` to `speakers`. Do they still work this way?

Reply:
I see the pictures as part of this diff were also added to `2019-speakers`. Can we move them into `speakers` with the rest of the speaker images?